iothub/service/src/IoTHubSasCredential.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+#if !NET451
+
+using Azure;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Shared access signature credential used to authenticate with IoT hub.
+    /// </summary>
+    public class IoTHubSasCredential : AzureSasCredential
```

Comment:
We have a mix of "IoTHub" vs "IotHub" casing in this project. From what I can tell, "IotHub" is a bit more common. Maybe match that casing here?

Reply:
It should be IotHub for classes. This one should be changed.